app-backend/common/src/main/scala/utils/Antimeridian.scala

```diff
@@ -0,0 +1,111 @@
+package com.azavea.rf.common.utils
+
+import geotrellis.proj4._
+import geotrellis.slick.Projected
+import geotrellis.vector._
+import geotrellis.vector.io._
+
+object AntimeridianUtils {
```

Comment:
Some explanation of the heuristics/checks here would be good so that future development knows if these functions can be used in new cases, need to be changed, or add new options.